impl/src/main/java/org/ehcache/impl/internal/statistics/DefaultTierStatistics.java

```diff
@@ -16,7 +16,13 @@
 
 package org.ehcache.impl.internal.statistics;
 
+import java.util.Collections;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Yeah... Ehcache is mainly putting java import at the top. But my IDE wasn't. I've fixed it and since I've created the class... I was in the wrong order.